source/api.py

```diff
@@ -317,8 +317,15 @@ def getStatusBar():
 	if not location:
 		return None
 	left, top, width, height = location
-	bottom = top + height - 1
-	obj = getDesktopObject().objectFromPoint(left, bottom)
+	x=1
+	while x<12:
+		bottom = top + height - x
+		if left<0:
+			left=0

```

Comment:
I assume this is because you were seeing cases where left was a negative number? When?